README.md

```diff
@@ -0,0 +1,217 @@
+any-migrate
+===========
+
+A generic data migrator.
+
+**Status**: Planning
+
+Rationale
+---------
+
+For proper continuous integration (CI) or continuous delivery (CD), many people
+use a database schema migration tool. The general idea is to versionize and
+codify migrations to make deployments repeatable, revertable, testable and
+automated.
+
+Database schema migrations tools generally supports versioning of schemas and
+applies schema changes to a database. They also keep track of every change
+applied to a system. Sadly, most schema migration tools are only dealing with
+SQL. This limits them to very basic migrations such as adding/deleting fields
+or updating a value of some field.
+
+Occasionally simple SQL executions aren't enough. The migration might be more
+complex than that and you want to execute a script. The script could clean up
+your database, execute `pt-schema-migrate` (for MySQL), do complex migrations
+of users, populate complex data, or other one-off tasks you'd like to execute.
+`any-migrate` is a tool aiming to support custom data migrations.
+
+`any-migrate` will have the following features:
+
+ * It keeps track of which migrations has been executed on a system.
+ * Fails as early as possible.
+ * Ask before assuming something. Explicitness is important. No magic.
```

Comment:
Agree on this. Think it is really important.  Think there should be an option to say "Y/N" before each migration.